experiments/huffpost/compare_trigger_policies/pipeline_config.py

```diff
@@ -0,0 +1,99 @@
+from modyn.config.schema.pipeline import (
+    CheckpointingConfig,
+    DataConfig,
+    EvalDataConfig,
+    EvalHandlerConfig,
+    EvaluationConfig,
+    FullModelStrategy,
+    ModelConfig,
+    ModynPipelineConfig,
+    NewDataStrategyConfig,
+    OptimizationCriterion,
+    OptimizerConfig,
+    OptimizerParamGroup,
+    Pipeline,
+    PipelineModelStorageConfig,
+    TrainingConfig,
+    TriggerConfig,
+)
+from modyn.config.schema.pipeline.evaluation.metrics import AccuracyMetricConfig, F1ScoreMetricConfig
+
+
+def gen_pipeline_config(
+    name: str, trigger: TriggerConfig, eval_handlers: list[EvalHandlerConfig]
+) -> ModynPipelineConfig:
+    num_classes = 42
+    return ModynPipelineConfig(
+        pipeline=Pipeline(name=name, description="Huffpost pipeline for comparing trigger policies", version="0.0.1"),
+        model=ModelConfig(id="ArticleNet", config={"num_classes": num_classes}),
+        model_storage=PipelineModelStorageConfig(full_model_strategy=FullModelStrategy(name="PyTorchFullModel")),
+        training=TrainingConfig(
+            gpus=1,
+            device="cuda:0",
+            dataloader_workers=8,
+            use_previous_model=True,
+            initial_model="random",
+            batch_size=64,
+            shuffle=True,
+            optimizers=[
+                OptimizerConfig(
```

Comment:
just a general note: for some benchmarks, I am still exploring the baseline (basically arXiv right now, but CLOC is also open). however this should not affect what you are doing. you can for sure use a different baseline than we might do for the sigmod submisison and then update later.